spec/ruby/library/socket/socket/gethostname_spec.rb

```diff
@@ -3,6 +3,6 @@
 
 describe "Socket.gethostname" do
   it "returns the host name" do
-    Socket.gethostname.should == `hostname`.strip
+    Socket.gethostname.should == `uname -n`.strip
```

Comment:
This will break on **pure** Windows (GitHub Actions have msys).

Reply:
Maybe we can have a conditional on Windows to use `hostname`?

Reply:
Yes it would be good to avoid using `uname` on Windows as it might not be there/it's not the right thing to use.
You can use `platform_is :windows do` for that

Reply:
I'll revert this, `hostname` is more portable, and Arch can add a dependency to `inetutils` to run specs

Reply:
`hostname` is not more portable, it is a GNU-specific tool (`inettools`) and `uname -n` is standardised by POSIX/Open Group: https://pubs.opengroup.org/onlinepubs/009695399/utilities/uname.html and should be available everywhere. However, I accept on vanilla Windows, only `hostname.exe` exists.